website/models.py

```diff
@@ -68,6 +68,56 @@ class Meta:
             ("Retired", "Previous researchers on project."),
         )
 
+class Dataset(models.Model):
+    #unique ids are automatically generated by django for every model instance
+    name = models.CharField(max_length=200, default="")
+
+    def __str__(self):
+        return self.name
+
+class ProjectSelector(models.Model):
+    input_dataset = models.IntegerField(blank=True)
+    input_selection = models.IntegerField(blank=True)
+    #When a filter is connected via foreign key django should automatically generate a set containing all connected filters that can be accessed
+    #via query
+    output = models.IntegerField()
+
+    def __str__(self):
+        return self.input_dataset + ' - ' + self.input_selection
+
+class Filter(models.Model):
+    project_selector = models.ForeignKey(ProjectSelector, on_delete=models.CASCADE)
```

Comment:
Isnt this backwards?  A filter can be used by many different project selectors.  This is a many-many relation.

Reply:
This is still backwards. ProjectSelector picks Filters. This is the other way.